crates/kitsune_p2p/kitsune_p2p/src/gossip/sharded_gossip.rs

```diff
@@ -552,7 +554,7 @@ impl ShardedGossipLocalState {
                 Some(when_initiated)
                     if no_current_round_exist && when_initiated.elapsed() > round_timeout =>
                 {
-                    tracing::error!("Tgt expired {:?}", cert);
+                    tracing::warn!("Gossip tgt expired {:?}", cert);
```

Comment:
and here

Reply:
updated to tracing::warn!("Gossip partner has timed out, considering them expired for the current round {:?}", cert);

how does that sound?

Reply:
nice!